<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogCamera</class>
 <widget class="QDialog" name="dialogCamera">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="1">
    <widget class="QSpinBox" name="valVFov">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> deg</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>120</number>
     </property>
     <property name="value">
      <number>90</number>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>LookFrom</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Z:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="valY">
        <property name="minimum">
         <number>-10</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="valZ_2">
        <property name="minimum">
         <number>-20</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="valZ">
        <property name="minimum">
         <number>-10</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="valX">
        <property name="minimum">
         <number>-10</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="valY_2">
        <property name="minimum">
         <number>-20</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="valX_2">
        <property name="minimum">
         <number>-20</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="4" rowspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>LookAt</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="2">
       <widget class="QSpinBox" name="valLookAtX_2">
        <property name="suffix">
         <string/>
        </property>
        <property name="minimum">
         <number>-20</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="valLookAtX">
        <property name="minimum">
         <number>-10</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="valLookAtY">
        <property name="minimum">
         <number>-10</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="valLookAtZ_2">
        <property name="minimum">
         <number>-20</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::DefaultStepType</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="valLookAtY_2">
        <property name="suffix">
         <string/>
        </property>
        <property name="minimum">
         <number>-20</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="valLookAtZ">
        <property name="minimum">
         <number>-10</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>vfov</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>valX</tabstop>
  <tabstop>valX_2</tabstop>
  <tabstop>valY</tabstop>
  <tabstop>valY_2</tabstop>
  <tabstop>valZ</tabstop>
  <tabstop>valZ_2</tabstop>
  <tabstop>valLookAtX</tabstop>
  <tabstop>valLookAtX_2</tabstop>
  <tabstop>valLookAtY</tabstop>
  <tabstop>valLookAtY_2</tabstop>
  <tabstop>valLookAtZ</tabstop>
  <tabstop>valLookAtZ_2</tabstop>
  <tabstop>valVFov</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dialogCamera</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dialogCamera</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>valX_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valX_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>valX</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>valY_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valY_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>valY</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>valZ_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valZ_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>valZ</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valLookAtX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>valLookAtX_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valLookAtX_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>valLookAtX</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valLookAtY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>valLookAtY_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valLookAtY_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>valLookAtY</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valLookAtZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>valLookAtZ_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valLookAtZ_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>valLookAtZ</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
